Sources/PubNub/Networking/Endpoints/Endpoint.swift

```diff
@@ -145,6 +154,7 @@ public enum Endpoint {
 
   // History
   case fetchMessageHistory(channels: [String], max: Int?, start: Timetoken?, end: Timetoken?, includeMeta: Bool)
+  case fetchMessageHistoryWithActions(channel: String, max: Int?, start: Timetoken?, end: Timetoken?, includeMeta: Bool)
```

Comment:
While this is a new API Endpoint in the backend, we had decided to just add a flag in `fetchMessageHistory` titled `includeMessageActions` that would then route to the different endpoint.

Reply:
I can collapse this (on the publish) into the existing method.

Going forward is the preference to add additional method parameters to existing methods when performing similar functions regardless of the difference in server endpoint (history vs history-with-actions) or amount of calls required (publish and add action)?

Reply:
This is somewhat of a special case because the main usage of the endpoint is almost exactly the same as the one before but includes message actions. The goal here was to make the usage as simple as possible for the developer since that is the main difference.

Reply:
Yeah that makes complete sense since everything is the same except that single path component.  I think for more complex cases we want to keep the complexity out of the endpoint, and shift it out elsewhere.